### ISSUE TITLE
ggsave: saving `geomImshow()` to SVG produces fuzzy picture

### ISSUE THREAD

**Issue by alshan** [JetBrains/lets-plot-kotlin#188]
Try `ggsave` the last plot in [image_101.ipynb](https://nbviewer.org/github/JetBrains/lets-plot-kotlin/blob/master/docs/examples/jupyter-notebooks/f-4.2.0/image_101.ipynb) to svg :

![image](https://github.com/JetBrains/lets-plot-kotlin/assets/4430922/2da2ccce-90a0-4234-b3a0-42db3558cfac)

Saving to HTML is fine.

**Comment by alshan** [comment]
Same issue in Python LP.

**Comment by alshan** [comment]
In prev. LP v3.1.0 (py) saving imshow to SVG worked producing sharp images.

**Comment by IKupriyanov-HORIS** [comment]
Test notebook:

```kotlin
%use lets-plot
%use lib-ext

LetsPlot.getInfo()

//%

import java.awt.image.*
import javax.imageio.*
import java.nio.file.*

fun display(path: String): Any {
    if (path.endsWith("svg")) {
        val svg = Files.readString(Path.of(path))
        return HTML(svg)
    } else {
        val img = ImageIO.read(File(path))
        return Image(img)
    }
}

//%

val A2x3x4 = RasterData.create(
    listOf(
        listOf(
            listOf(1.0, 0.0, 0.0, 1.0), listOf(0.0, 1.0, 0.0, 1.0), listOf(0.0, 0.0, 1.0, 1.0)
        ),
        listOf(
            listOf(0.0, 1.0, 0.0, 0.3), listOf(0.0, 0.0, 1.0, 0.3), listOf(1.0, 0.0, 0.0, 0.3)
        )
    )
)

//%

val p = ggplot() + geomImshow(A2x3x4)
p.show()

//%

display(ggsave(p, "LPK-188.svg"))

//%

display(ggsave(p, "LPK-188.png"))
```

**Comment by alshan** [comment]
Fixed in 4.4.2
